package.json

```diff
@@ -4,8 +4,9 @@
   "private": true,
   "scripts": {
     "start": "node ./bin/www",
-    "dev": "nodemon ./bin/www & npm run scss",
-    "scss": "sass --watch src/scss/main.scss public/stylesheets/style.css"
+    "dev": "nodemon ./bin/www & npm run scss & npm run js",
+    "scss": "sass --watch src/scss/main.scss public/stylesheets/govuk-frontend.min.css --style=compressed",
```

Comment:
We may not name our compiled css as 'govuk-frontend'. Project specific scss will be in main.scss as well.

Reply:
Agreed, maybe the generic (application.min.css etc) is better than saying GOVUK.